### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by judyzhang25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,134 +1,38 @@
...
-    > Success Condition:
-    > ```
+# Dronuts
+### Ryan Barrett, Derek Yan, Judy Zhang
+This is Group 3's project for CMU's Software Engineering for Startups class (17-356). This readme will describe how to run and test the Dronuts application.
+
+## How to run the application for...
+### Development
+
+To run the application in development mode, run the following code from the root directory:
```
Comment (derekyyyan) [inline README.md]: I believe `npm start` should be run in `/Dronuts` instead of the root directory

Comment (derekyyyan) [reply README.md]: Should probably mention running `npm install` too

File: README.md [hunk 2]
```diff
@@ -1,134 +1,38 @@
...
+### Production
+
+To run the application in production mode, we first need to build a docker container before running our code. The following code demonstrates how to do so:
+
+    > $ docker-compose build
+    > $ docker-compose up
+
+## Testing with Travis CI
+
+TravisCI runs with the script found in .travisyml from the root directory. It runs automatically when changes are pushed to Git.
```
Comment (derekyyyan) [inline README.md]: should be `.travis.yml`

File: README.md (hunk 1 above)
Comment (judyzhang25) [reply README.md]: Will make the changes

File: README.md (hunk 2 above)
Comment (judyzhang25) [reply README.md]: Got it

**Comment by derekyyyan** [comment]
lgtm

**Pull request merged**